src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/indexed_fields_table/components/table/table.js

```diff
@@ -41,9 +44,12 @@ export class Table extends PureComponent {
         {isConflict ? (
           <span>
             &nbsp;
-            <EuiToolTip content="The type of this field changes across indices. It is unavailable for many analysis functions.">
-              <EuiIcon type="alert" color="warning" />
-            </EuiToolTip>
+            <EuiIconTip
+              type="alert"
+              color="warning"
+              aria-label="Warning"
```

Comment:
Same as above, I would suggest using something like "Multiple type field" instead of "Warning".

Reply:
makes sense! updating